packages/frontend/src/lib/pnl.ts

```diff
@@ -85,7 +86,11 @@ export async function calcETHCollateralPnl(
       }, Promise.resolve({ deposits: BIG_ZERO, withdrawals: BIG_ZERO, priceError: false }))
     : { deposits: BIG_ZERO, withdrawals: BIG_ZERO, priceError: true }
 
-  return !priceError ? currentVaultEthBalance.times(uniswapEthPrice).minus(deposits.minus(withdrawals)) : BIG_ZERO
+  if (priceError) {
+    throw new Error('Collateral PnL Error')
```

Comment:
@adam5909099 Can we know exactly why the error occurs? Like tweledata API failiure, etc